src/components/setup-authenticator-app/index.njk

```diff
@@ -47,7 +48,7 @@
   hint: {
       text: 'pages.setupAuthenticatorApp.code.hintText' | translate
   },
-  classes: "govuk-!-width-two-thirds govuk-!-font-weight-bold",
+  classes: "govuk-input--width-10 govuk-!-font-weight-bold",
```

Comment:
Assume this is harmonised with other places now (I quickly checked `enter-authenticator-app-code` view, which is the only other place I know this pops up)

Reply:
We need to discuss this with Content.